File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
// class Solution {
//     public int removeDuplicates(int[] nums) {
//         Set<Integer>st=new LinkedHashSet<>();
//         for(int n:nums){
//             st.add(n);
//         }
//         int i=0;
//         for(int val:st){
//             nums[i]=val;
//             i++;
//         }
//         return st.size();
//     }
// }
class Solution {
    public int removeDuplicates(int[] nums) {
        int i=0;
        for(int j=1;j<nums.length;j++){
            if(nums[i]!=nums[j]){
                nums[++i]=nums[j];
            }
        }
        return i+1;
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        # unique=sorted(set(nums))
        # for i in range(len(unique)):
        #     nums[i]=unique[i]
        # return len(unique)
        i=0
        for j in range(1,len(nums)):
            if(nums[i]!=nums[j]):
                i+=1
                nums[i]=nums[j]
        return i+1